sw/device/lib/runtime/ibex.h

```diff
@@ -33,7 +33,7 @@ inline void ibex_busy_loop(size_t cycles) {
       "busy_loop%=:"
       "  nop; nop; nop; nop;"    // 4 cycles.
       "  addi %1, %1, -8;"       // 1 cycle.
-      "  blez %1, busy_loop%=;"  // 3 cycles.
+      "  bgez %1, busy_loop%=;"  // 3 cycles.
```

Comment:
Oops, I think I was the person that said it should be `blez`. Sorry for the confusion.

Reply:
No worries and thanks for the clarification about the data types on rv32!